models/repos.js

```diff
@@ -11,6 +11,11 @@ const assert = require('assert')
 const mkdirp = require('mkdirp')
 const open = require('open')
 const path = require('path')
+const mock = require('../tests/mocks')
+
+if (process.env.RUNNING_IN_SPECTRON) {
+  mock(dialog)
+}
```

Comment:
Makes sense, I'm not sure how to make this better though. If you have an idea that'd be useful but I think that i'd rather require something from tests than patch require.